open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/toolbox/SimpleRegistryImpl.java

```diff
@@ -0,0 +1,44 @@
+package io.opensphere.controlpanels.toolbox;
+
+import java.util.Collection;
+import java.util.Properties;
+
+import io.opensphere.controlpanels.SimpleRegistry;
+import io.opensphere.core.Toolbox;
+import io.opensphere.core.util.collections.New;
+
+/**
+ * A default implementation of the {@link DetailPanelProviderRegistry}, in which
+ * providers are cached.
+ * 
+ * @param <T>
+ */
+public class SimpleRegistryImpl<T> implements SimpleRegistry<T>
+{
+    /**
+     * The collection of registered providers.
+     */
+    private Collection<T> myProviders;
+
+    @Override
+    public void registerProvider(T provider)
+    {
+        myProviders.add(provider);
+
```

Comment:
Remove the extra blank line (you can get Eclipse to remove extra lines for you on save)